webapp-mgmt/cas-management-webapp-configuration/src/main/java/org/apereo/cas/mgmt/configuration/CasConfigurationPropertiesSourceLocator.java

```diff
@@ -0,0 +1,174 @@
+package org.apereo.cas.mgmt.configuration;
+
+import org.apache.commons.io.FileUtils;
```

Comment:
This file need to go away. You should use one that is provided to you by the configuration-api module.